### ISSUE TITLE
weights loading has something wrong

### ISSUE THREAD

**Issue by sanmianjiao** [aloyschen/tensorflow-yolo3#8]
I tried to train this model without darknet53.weights. when I finish 5 Epoch with model saved, I found that I can't load correct weights from my old trained model weights . The loss start from 0, and rise always , can you tell me why.
我没用darknet53做预训练，直接跑的coco数据集，保存了一个阶段成果之后再加载我之前训练的参数，发现损失值loss从0开始 一直往上增加 ，请问这个现象是为什么呢？

**Comment by 760146355** [comment]
我也遇到这个问题了，请问你解决了吗

**Comment by jjwwczy** [comment]
因为打印出来的loss实际上并不是当前batch的loss，而是和之前的累计loss以及gloabal_steps有关的。重新开始以后gloabal_steps和总的loss都重新开始计数了。 可以考虑修改语句，把当前batch的loss打印出来。
```
format_str = ('Epoch {} step {},  train loss = {} ( {} examples/sec; {} ''sec/batch) train_loss={}')
print(format_str.format(epoch, step, loss_value / global_step_value, examples_per_sec, duration,train_loss))
```
